### ISSUE TITLE
Javascript documentation on how to include other libraries

### ISSUE THREAD

**Issue by skully78** [vigetlabs/gulp-starter#281]
Hi. Digging this gulp-starter. I am looking for a way to combine all the JavaScript bower components into one file for the project.  Is there a way to do it with the current webpack-multi-config? Or could you provide more documentation on how to properly achieve this please. Thank you.

**Comment by Necmttn** [comment]
+1 up up

**Comment by greypants** [comment]
Is this helpful? https://webpack.github.io/docs/usage-with-bower.html

I've never used Bower, but I suppose you could. Any reason not to use NPM js modules instead?

Also, https://github.com/vigetlabs/gulp-starter/issues/128 may be relevant.

**Comment by medinarodel** [comment]
Need some help. I tried installing MaterializeCSS via npm (http://materializecss.com/getting-started.html), it was successful. I would like to ask how can I use that in the app I'm working? MaterializeCSS js and CSS files are not present in public folder.

I don't think this is the right way and I am only uploading the public folder to our server.
```
<script type="text/javascript" src="node_modules/materialize-something/materialize.min.js"></script>
```


thanks!

**Comment by greypants** [comment]
@medinarodel 

In your main js file, 
```
@import 'materialize'
```
And for your css, add materilize to your included paths [like I'm doing with normalize]( (https://github.com/vigetlabs/gulp-starter/blob/master/gulpfile.js/config.json#L40)), then in your main sass file, 

```
@import 'materialize'
```

**Comment by medinarodel** [comment]
@greypants 

Thanks for your response. 

I just can't still figure it out. 

When you say "main js" file, are you referring to the file where I will use materialize? I tried adding `@import 'materialize'` but I am getting an error unexpected token `'@'`, so I tried `import 'materialize'` but got an error `Cannot resolve module 'materialize'`

And for the css, where is the included paths? Sorry about being noob.

I also tried using other plugins like Lightbox2(http://lokeshdhakar.com/projects/lightbox2/). I installed it via npm. I was able to use the JS and CSS of that plugin by doing this:
For JS, I added this on the js file I need it:
`import * as lightbox from '../../../../node_modules/lightbox2/dist/js/lightbox.js'`

For CSS, I added this
`@import '../../node_modules/lightbox2/dist/css/lightbox'`. However, I need to manually copy the images to `src/images`.
